ocpp/v16/call.py

```diff
@@ -229,3 +234,72 @@ class DataTransferPayload:
     vendor_id: str
     message_id: Optional[str] = None
     data: Optional[str] = None
+
+
+# Backported CALL Messages
+# Improved security for OCPP 1.6-J specification
+
+@dataclass
+class CertificateSignedPayload:
+    cert: List
```

Comment:
The schemas for this request don't match the specification:

![Screenshot_2021-04-26_11-16-26](https://user-images.githubusercontent.com/1565144/116059259-f0bee880-a680-11eb-97d0-16714d2593e0.png)

My suggestion is to follow the written specification. Please use `certificate_chain` in both here and the schema's.

Reply:
Make sure to update the type from `List` to `str` as well.